src/webview/git-import/app/cardItem.tsx

```diff
@@ -0,0 +1,55 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Card, makeStyles, Typography } from '@material-ui/core';
+import cardItemStyle from './cardItem.style';
+
+interface CardProps extends React.AllHTMLAttributes<HTMLDivElement> {
+    yamlDoc: any;
+}
+
+const makeCardStyle = makeStyles(cardItemStyle);
+
+export const CardItem: React.FC<CardProps> = ({
+    yamlDoc
+}: CardProps) => {
+
+    const style = makeCardStyle();
+
+    return (
+        <>
+            <Card
+                className={style.card}
+                data-testid={`card-${yamlDoc.metadata.name.replace(/\.| /g, '')}`}
+            >
+                <div className={style.cardHeader}>
+                    <div className={style.cardHeaderDisplay}>
+                        <img
+                            src={yamlDoc.metadata.icon}
+                            className={style.cardImage} />
+                    </div>
+                </div>
+                <div style={{ margin: '1.5rem' }}>
+                    <Typography variant='body1'>{yamlDoc.metadata.name}</Typography>
```

Comment:
Here also we need to keep it full name. Let's see if we can map `metadata.name` to use fullName.